### PR TITLE
Update aws images url endpoints

### PR DESCRIPTION
**Pull request by YuvalFeldman**

### CODE REVIEW

File: charts/tensorleap/charts/engine/templates/engine-warmup-job.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ data:
             spec:
                 containers:
                 - name: engine-placeholder
-                  image: us-central1-docker.pkg.dev/tensorleap/main/engine:{{ .Values.image_tag }}
+                  image: public.ecr.aws/g9h8h3m5/engine:{{ .Values.image_tag }}
                   command: ["bash"]
```
Comment (moshe-kabala) [inline charts/tensorleap/charts/engine/templates/engine-warmup-job.yaml:18]: use {{ .Values.image }}

File: config/k3d-config.yaml [hunk 2]
```diff
@@ -61,6 +61,9 @@ registries:
       us-central1-docker.pkg.dev:
```
Comment (moshe-kabala) [inline config/k3d-config.yaml:61]: you can delete this file

File: pkg/k3d/utils.go [hunk 3]
```diff
@@ -7,32 +7,6 @@ import (
...
-//	docker.elastic.co:
-//	  endpoint:
-//	    - http://k3d-tensorleap-registry:5000
-//	quay.io:
-//	  endpoint:
-//	    - http://k3d-tensorleap-registry:5000
-//	us-central1-docker.pkg.dev:
-//	  endpoint:
-//	    - http://k3d-tensorleap-registry:5000
-//	        `
```
Comment (moshe-kabala) [inline pkg/k3d/utils.go:35]: return back

**Review by moshe-kabala — APPROVED** [review]
LGTM
You need to require the user to reinstall - you can do it by changing the chart version to minor (e.g 0.0.0 0.1.0) changing the middle number

File: config/k3d-config.yaml (hunk 2 above)
Comment (YuvalFeldman) [reply config/k3d-config.yaml:61]: not right now

**Pull request merged**